manager/state/raft/raft_test.go

```diff
@@ -226,7 +227,9 @@ func TestRaftQuorumRecovery(t *testing.T) {
 	// Restore the majority by restarting node 3
 	nodes[3] = raftutils.RestartNode(t, clockSource, nodes[3], false)
 
+	raftutils.ShutdownNode(nodes[1])
 	delete(nodes, 1)
+	raftutils.ShutdownNode(nodes[2])
```

Comment:
but those nodes already stopped above and they're deleted from cluster(which is just map here)